ast/astVisit.ml

```diff
@@ -392,3 +392,24 @@ let collect_accesses_in_position statement { Location.line; column } =
         | _ -> None
     end) in
   Collector.collect (Source.create [statement])
+
+
+let contains_yield {Statement.Define.body; _} =
```

Comment:
I tried that, but I couldn't get it too work because analysisTypeCheck.ml already uses the Annotation name for the file analysisAnnotation.ml. Do you mind if I put the function there instead?